PredixMobileUIKitTests/PredixBarChartViewTests.swift

```diff
@@ -0,0 +1,369 @@
+    }
+
+    // MARK: PredixBarCharViewMethods
+
+    func testInitWithCoder() {
+        let barChart = PredixBarChartView()
+        let data = NSMutableData()
+        let coder = NSKeyedArchiver(forWritingWith: data)
+        coder.encode(barChart)
+        coder.finishEncoding()
+
+        let decoder = NSKeyedUnarchiver(forReadingWith: data as Data)
+        let newBarChart = PredixDonutView(coder: decoder)
+        XCTAssertNotNil(newBarChart)
+    }
+
+    func testInitWithFrame() {
+        let rect = CGRect(x: 0.0, y: 0.0, width: 0.6, height: 0.4)
+        let barChart = PredixBarChartView(frame: rect)
+        XCTAssertEqual(0.0, barChart.frame.origin.x, accuracy: 0.01)
+        XCTAssertEqual(0.0, barChart.frame.origin.y, accuracy: 0.01)
+        XCTAssertEqual(0.6, barChart.frame.size.width, accuracy: 0.01)
+        XCTAssertEqual(0.4, barChart.frame.size.height, accuracy: 0.01)
+    }
+
+    // MARK: PredixBarCharView HandleOptionMethods
+
+    func testHandleOptionEnableLegend() {
+        let barChart = PredixBarChartView()
+        barChart.handleOption(.enableLegend)
+        XCTAssertEqual(barChart.legend.enabled, true, "Option enableLegend should be true")
+    }
+
+    func testHandleOptionDisableLegend() {
+        let barChart = PredixBarChartView()
+        barChart.handleOption(.disableLegend)
+        XCTAssertEqual(barChart.legend.enabled, false, "Option disableLegend should be false")
+    }
+
+    func testHandleOptionDisableSideLabels() {
+        let barChart = PredixBarChartView()
+        barChart.handleOption(.disableSideLabels)
+        XCTAssertEqual(barChart.rightAxis.drawLabelsEnabled, false, "Right Axis drawLabelsEnabled should be false")
+        XCTAssertEqual(barChart.leftAxis.drawLabelsEnabled, false, "Left Axis drawLabelsEnabled should be false")
+    }
+
+    func testHandleOptionEnableSideLabels() {
+        let barChart = PredixBarChartView()
+        barChart.handleOption(.enableSideLabels)
+        XCTAssertEqual(barChart.rightAxis.drawLabelsEnabled, true, "Right Axis drawLabelsEnabled should be true")
+        XCTAssertEqual(barChart.leftAxis.drawLabelsEnabled, true, "Left Axis drawLabelsEnabled should be true")
+    }
+
+    func testHandleOptionToggValues() {
+        let barChart = PredixBarChartView()
+        var months: [String]!
+        months = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"]
+        let unitsSold = [20.0, 4.0, 6.0, 3.0, 12.0, 16.0, 4.0, 18.0, 2.0, 4.0, 5.0, 4.0]
+        let unitsBought = [10.0, 2.0, 20.0, 4.0, 5, 8.0, 9.0, 15.0, 1.0, 3.0, 10.0, 18.0]
+
+        let bar1 = Bar(unitsSold, label: "Units Sold", colors: [NSUIColor.blue])
+        let bar2 = Bar(unitsBought, label: "Units Bought", colors: [NSUIColor.red])
+
+        barChart.create(xAxisValues: months, bars: [bar1, bar2], showWithDefaultAnimation: true)
+        XCTAssertTrue(barChart.data?.dataSetCount ?? 0 > 0, "No datasets were loaded in prepareForInterfaceBuilder")
```

Comment:
Error message incorrect, there is no prepareForInterfaceBuilder in the PredixBarChartView.

Reply:
Remove the prepareForInsterfaceBuilder line